tests/baselines/reference/ES3For-ofTypeCheck1.errors.txt

```diff
@@ -1,7 +0,0 @@
-tests/cases/conformance/statements/for-ofStatements/ES3For-ofTypeCheck1.ts(1,15): error TS2494: Using a string in a 'for...of' statement is only supported in ECMAScript 5 and higher.

```

Comment:
Should these "ES3For" tests be modified to still test ES3, but explicitly?

Reply:
I guess maybe it doesn't matter if the next PR is "delete ES3"

Reply:
We should do a post-pass to clean up any irrelevant tests